### PR TITLE
CR-1051302 Add endpoint name to dmesg log on firewall trip

### PR DESCRIPTION
**Pull request by xdavidz**

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c [hunk 1]
```diff
@@ -292,7 +292,9 @@ static u32 check_firewall(struct platform_device *pdev, int *level)
 				xocl_ioaddr_to_baroff(xdev, res->start,
 					&bar_idx, &bar_off);
 			}
-			xocl_info(&pdev->dev, "AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx", i, val, bar_off);
+			/* To be improved: link IORESOURCE_MEM(0) with NODE_AF_BLP string */
+			xocl_info(&pdev->dev, "AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx, resource name:%s",
+			    i, val, bar_off, NODE_AF_BLP);
```
Comment (houlz0507) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c]: the endpoint name is based on level. It should not use a fixed string here. And the resource name could be NULL for legacy shell.

Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c]: Thanks. I will make the changes.

Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c]: @houlz0507 would this version be better?

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c [hunk 2]
```diff
@@ -293,8 +293,8 @@ static u32 check_firewall(struct platform_device *pdev, int *level)
 					&bar_idx, &bar_off);
 			}
 			xocl_info(&pdev->dev,
-				"AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx",
-				i, val, bar_off);
+				"AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx, resource %s",
+				i, val, bar_off, res->name ? res->name : "N/A");
 			if (!fw->curr_status) {
```
Comment (xdavidz) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c]: @houlz0507 would this version be better?

**Pull request merged**